cl/lib/elasticsearch_utils.py

```diff
@@ -7,29 +7,46 @@
 from dataclasses import fields
 from datetime import date, datetime
 from functools import reduce, wraps
-from typing import Any, Callable, Dict, List, Literal, Match
+from typing import Any, Callable, Dict, List, Literal, Tuple
```

Comment:
We should remove the `Tuple` import, we can use the built-in collection type `tuple` as a generic type instead

Reply:
I haven't been mentioning this, but we should be doing this any time we're adding type hints. I look forward to the day when all of the type imports are removed.